notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Typo:&nbsp;a manifold can be defined as an open set of a d-dimensional vector space</p><p> </p><p>(the manifold IS the open set)</p><p> </p><p>Typo: intuitively</p><p> </p><p>NIT: would</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>done</p>